tests/ext/span_with_parent_property_access.phpt

```diff
@@ -15,10 +15,6 @@ if (!isset($root->parent) && $span->parent === $root) {
         $span->parent = null;
     } catch (\Exception $error) {
         print $error->getMessage() . PHP_EOL;
-    } finally {
-        if ($span->parent === null) {
```

Comment:
btw. you can keep that if, given that the catch block doesn't rethrow anyway, just dropping finally {}

Reply:
5.5 is first version with finally {}